setup-cluster.sh

```diff
@@ -13,6 +13,19 @@ NODE_BOX="centos/7"
 MASTER_BOX="centos/7"
 CACHE_FOUND=false
 
+function clone-openshift-ansible {
+  if [ ! -d "openshift-ansible" ] ; then
+    echo "Cloning openshift-ansible..."
+    git clone "https://github.com/ewolinetz/openshift-ansible.git"
```

Comment:
Personally, I'm more of a fan of documenting this instead of cloning it. Or making an error message if it doesn't exist. But, I'm not going to block on this.